api/v1alpha1/schedule_webhook_test.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2021 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("schedule_webhook", func() {
+	Context("webhook.Validator of schedule", func() {
+		It("Validate", func() {
+
+			type TestCase struct {
+				name     string
+				schedule Schedule
+				execute  func(schedule *Schedule) error
+				expect   string
+			}
+			tcs := []TestCase{
+				{
+					name: "validation for normal chaos",
+					schedule: Schedule{
+						ObjectMeta: metav1.ObjectMeta{
+							Namespace: metav1.NamespaceDefault,
+							Name:      "foo1",
+						},
+						Spec: ScheduleSpec{
+							ScheduleItem: ScheduleItem{EmbedChaos: EmbedChaos{PodChaos: &PodChaosSpec{
+								Action: ContainerKillAction,
+							}}},
+							Type:     ScheduleTypePodChaos,
+							Schedule: "@every 5s",
+						},
+					},
+					execute: func(schedule *Schedule) error {
+						return schedule.ValidateCreate()
+					},
+					expect: "error",
```

Comment:
Why `"@every 5s"` is invalid?

Reply:
`containerNames` shouldn't be empty if the `action` is "container-kill"